### ISSUE TITLE
Reconcile rendering stats & rendering frame with chrome helper

### ISSUE THREAD

**Issue by nedn** [catapult-project/catapult#1491]
rendering_stats & rendering_frame was telemetry's helper modules for computing smoothness metrics. Once #1228 is done, we should merge these helper module to chrome helper.

**Comment by nedn** [comment]
Won't fix since we may use RAIL's metrics to replace smoothness.
